### PR TITLE
Make project setup

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request, I will add project setup files, also I will add header, banner, and I will style it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css">
+</head>
+<body>
+
+    <!--HEADER-->
+    <header>
+        <div class="logo">
+            <a href="index.html">
+                <img src="mylogo.png" alt="My Logo">
```
Comment (descholar-ceo) [inline index.html:16]: - [ ] This image is broken, would you fix it?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                    <a href="">
+                        <img src="images/mIcon.svg" alt="m-icon">
+                    </a>
+                    <a href="">
+                        <img src="images/gitHubIcon.svg" alt="github-icon">
+                    </a>
+                    <a href="">
+                        <img src="images/redIcon.svg" alt="red-icon">
+                    </a>
+                </div>
```
Comment (descholar-ceo) [inline index.html:53]: - [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @vladanvi99 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] The screenshot from the readme file is broken, would you fix it?

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (vladanvi99) [reply index.html]: Please check now, I fixed all.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @vladanvi99 , 

### Status: Approved ✔️

:+1:  Professional repo, with good commit message and PR title, keep up the good work :muscle:
:+1: The project setup is well organized. Well Done.

Good job on implementing the suggested changes. Well Done.

Your project is complete!  There is nothing to say other than ... it's time to merge it.
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**